ambari-server/src/test/java/org/apache/ambari/server/controller/internal/MpackResourceProviderTest.java

```diff
@@ -112,7 +111,9 @@ public void testGetResourcesMpacks() throws Exception {
     // replay
     replay(m_dao);
 
-    ResourceProvider provider = AbstractControllerResourceProvider.getResourceProvider(type, m_amc);
+    ResourceProvider provider = AbstractControllerResourceProvider.getResourceProvider(
+            type
```

Comment:
```
[ERROR]   MpackResourceProviderTest.testGetResourcesMpacks:114 » NullPointer
```

Reply:
Hi,
So far I am not able to repro this error. I get 
Caused by: java.lang.ClassNotFoundException: org.apache.hadoop.metrics2.sink.timeline.TimelineMetric
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
I will look into this test class in my subsequent patch. Thanks